framework/PageActions/PageActionSwitch.tsx

```diff
@@ -42,6 +42,7 @@ export function PageActionSwitch<T extends object>(props: {
             label={label}
             labelOff={labelOff}
             isChecked={isChecked}
+            hasCheckIcon={true}
```

Comment:
Is this needed? I already see the check icon on action of type PageActionType.Switch

Reply:
@vidyanambiar - the flag is necessary for the top of the page action swicth - as seen in the screenshot included in the PR's description.